Resources/doc/reference/configuration/cache-manager.rst

```diff
@@ -39,6 +39,10 @@ your own service that implements ``FOS\HttpCache\ProxyClientInterface``.
         cache_manager:
             custom_proxy_client: acme.caching.proxy_client
 
+When you specify a custom proxy client, the bundle does not know about the
+capabilities of the client. The ``generate_url_type`` defaults to true and
+:doc:`tag support <tags>` is only active if explicitly enabled.
+
 ``generate_url_type``
```

Comment:
The constants of `UrlGeneratorInterface` changed in symfony 3 to integer instead of string/bool. So the documentation about those enum values is not correct anymore.